cypress_test/Makefile.am

```diff
@@ -106,8 +106,7 @@ MULTIUSER_TESTS_DONE= \
 check-local: do-check
 	$(if $(wildcard $(ERROR_LOG)),$(error CypressError: some tests failed!))
 
-# TODO: enable other tests $(MOBILE_TEST_FILES_DONE) $(IDLE_TEST_FILES_DONE) $(MULTIUSER_TESTS_DONE)
```

Comment:
Shall we keep ToDo for multiuser tests?
"MULTIUSER_TESTS_DONE"

Reply:
I see, ooks like multiuser tests failed for this PR but it may be a temp one i guess.